static/src/styles/core/color.styl

```diff
@@ -1,12 +1,15 @@
 $Color__light = #F2F2F2;
+$Color__medium = #878787;
```

Comment:
Any ideas for this? 'mediumGrey'? 'mediumBrightnessMuted'? 'fadedTextForLightBG'